phantomjs-env/src/main/scala/org/scalajs/jsenv/phantomjs/PhantomJSEnv.scala

```diff
@@ -133,6 +133,8 @@ final class PhantomJSEnv(config: PhantomJSEnv.Config) extends JSEnv {
     try {
       out.write(
           s"""// Scala.js Phantom.js launcher
+             |if (!console.error)
+             |  console.error = console.log;
```

Comment:
Perhaps mention the issue that this fixes, for example as follows:
```suggestion
             |if (!console.error)
             |  console.error = console.log; // #47
```

Reply:
Maybe we should just do this locally, so it's side effect free?

Reply:
Disregard my previous comment. I thought the problem is in the calls to `console.error` lower in this code.